userguide/snippets/es60.rst

```diff
@@ -126,8 +131,8 @@ secures the rail in place. Repeat this process on the other side.
 Mount Rack Rails
 ^^^^^^^^^^^^^^^^
 
-Install four cage nuts in the rack, one for each bracket. The lock
-screw for the bracket is already installed in the cage nut. Place the
+Install the four cage nuts with locking screws in the rack. Place one
+cage nut for each bracket, where the rail will attach. Place the
```

Comment:
The comma is really awkward here.  The rails can have four holes, so we should be specific about which one is used.  With those changes, we should be able to avoid future tense.